content/en/docs/setup/install/gateway/index.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Installing Gateways
+description: Install and customize Istio Gateways.
+weight: 40
+keywords: [install,gateway,kubernetes]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+Along with creating a service mesh, Istio allows managing [gateways](/docs/concepts/traffic-management/#gateways),
+which run at the edge of the mesh and allow fine-grained control over traffic entering and leaving the mesh.
+
+Follow this guide to deploy one or more gateways.
+
+## Prerequisites
+
+This guide requires the Istio control plane [to be installed](/docs/setup/install/) before proceeding.
+
+## Installing the gateway with injection
```

Comment:
Ah, I see this is a TODO in your description above. My suggestion:

Add a section, before this one, that talks about "Gateway installation options". You could include the material from the doc I referenced above, and then also discuss the shortcomings and that template injection is a better approach. That would be followed by this section, which is an implied recommended approach, even if we don't say that explicitly yet, since the feature is still alpha.

Reply:
> I see what you mean but this seems similar to multicluster, which the comments apply to equally, so it may be good to make them consistent? Seems like either both should be under install/, or they should both be split?

I tend to agree that this is not the right place for this doc. The difference is that every other doc actually installs Istio (i.e., starts 1 or more control planes). This is really part of configuring the mesh. The external control plane doc really makes this clear by describing the gateway install steps under "mesh admin steps", while the the rest of the Istio setup is the "mesh operator steps". (As an aside, for the same reason, I think that maybe the External Control Plane installation doc should be moved here.)

That said, I tend to think that maybe the best place for this is under "More Guides". It really is important, so I also think that every other install doc should have one or more clear references to this guide, so users don't miss it.

Reply:
Also, if we move this to the "More Guides" section, we could also add a reference on the _index.md page, i.e., after `Choose the guide that best suits your needs and platform.`, add another sentence something like `To install gateways in a mesh, refer to [Installing Gateways](link) for details`.

Reply:
https://github.com/istio/istio.io/pull/9727

Reply:
I moved it under more guides